pylint/checkers/utils.py

```diff
@@ -876,7 +876,11 @@ def uninferable_final_decorators(
             except AttributeError:
                 continue
         elif isinstance(decorator, nodes.Name):
-            import_node = decorator.lookup(decorator.name)[1][0]
+            lookup_values = decorator.lookup(decorator.name)
+            if lookup_values[1]:
+                import_node = decorator.lookup(decorator.name)[1][0]
+            else:
+                continue
```

Comment:
It look like this is never used, maybe we can remove it ? Beside right now nothing is exactly the same behavior, right ?

```suggestion
```

Reply:
This is used on 3.6 and 3.7. Coveralls is reported on 3.8 (I believe) that's why the coverage dropped. I have added a `pragma: no cover`.